content/en/docs/release-notes/release-notes-1.4.md

```diff
@@ -34,6 +32,247 @@ We have deprecated the following cert-manager APIs:
+uses the certificate at the start of the chain as the `ca.crt`.
+
+Note that this `ca.crt` may be an intermediate signing CA instead of the actual
+root CA.
+
+The reason why this feature did not exist previously is that finding which
+certificate is at the top of the chain is not trivial. Take a look at
+[@JoshVanL](https://github.com/JoshVanL)'s excellent
+[`ParseSingleCertificateChain`](https://github.com/jetstack/cert-manager/blob/5e2a6883c1202739902ac94b5f4884152b810925/pkg/util/pki/parse.go#L167-L244)
+to see what this is all about!
+
+### Helm chart: webhook externally accessible for bare-metal
+
+In [some](https://github.com/kubernetes/kubernetes/issues/72936#issue-399522387)
+Kubernetes setups, the apiserver is not able to talk to kube-dns (i.e., when
+Kubernetes is running on bare-metal with no special `resolv.conf`).
+
+To work around that, the cert-manager webhook can
+[now](https://github.com/jetstack/cert-manager/pull/3876) be configured to be
+accessible from outside of the cluster. For example, in `values.yaml`:
+
+```yaml
+# values.yaml
+webhook:
+  serviceType: LoadBalancer
+  loadBalancerIP: 85.78.90.100
+  url:
+    host: 85.78.90.100
+```
+
+### Helm chart: Service labels
+
+The cert-manager controller Service
+[now](https://github.com/jetstack/cert-manager/pull/4009) supports custom labels
+using the top-level field in `values.yaml`:
+
+```yaml
+# values.yaml
+serviceLabels:
+  app: armada-api
+```
+
+This may be useful in conjunction with Prometheus' `labelmap`. For example, with
+the following sample Prometheus configuration:
+
+```yaml
+# prometheus.yaml
+- action: labelmap
+  regex: __meta_kubernetes_service_label_(.+)
+```
+
+With the above example, the source label
+`__meta_kubernetes_service_label_app='armada-api'` becomes the new label
+`app='armada-api'` when metrics related to this Service are scraped.
+
+## Akamai issuer and Open Edgegrid EdgeDNS v2
+
+The Akamai issuer has been
+[updated](https://github.com/jetstack/cert-manager/pull/4007) to use Open
+Edgegrid EdgeDNS v2 API.
+
+# Bug Fixes
+
+- The default certificate renewal duration of 30 days was clashing with the
+  duration of certificates issued by the Vault PKI. All Certificates [are
+  now](https://github.com/jetstack/cert-manager/pull/4092) renewed 2/3 through
+  the duration unless a custom renew period is specified with the setting
+  `spec.renewBefore` on the Certificate.
+- The [RFC2136](https://cert-manager.io/docs/configuration/acme/dns01/rfc2136/)
+  issuer is [now](https://github.com/jetstack/cert-manager/pull/3622) able to
+  handle DNS-01 challenges that map to multiple `TXT` records. This lets you
+  create Let's Encrypt certificates using RFC2136 with multiple DNS names.
+- The comparison function `PublicKeysEqual` is
+  [now](https://github.com/jetstack/cert-manager/pull/3914) correct for public
+  keys.
+- The upgrade from v1.3.0 to 1.4.0 with Helm and `--set installCRDs=true` is
```

Comment:
This was done so it's possible to upgrade from `v1.2.x` to  `v1.3.x`. I think this might not be relevant for `v1.4`?

Reply:
Should I rephrase as

```suggestion
- The upgrade from 1.2 to 1.3 with Helm and `--set installCRDs=true` is
```

Reply:
> I have tested v1.3.0 -> latest master (so with this PR) upgrade on kind clusters with k8s v1.20 and v1.19 (with helm)

The goal of the PR was to fix a bug where users couldn't upgrade _to_ `v1.3.x`. The other comments are mostly for edge cases. We also didn't merge this PR into `v1.3.0`.

Reply:
Wait, so this PR is only meant to fix e.g. `v1.2.0` -> `v1.3.1` but we did not backport it to `release-1.3`?

Reply:
I shall then create a new `1.3.1` entry in the 1.3 release notes page 😅